gulpfile.babel.js

```diff
@@ -34,7 +36,13 @@ gulp.task('lint', () => {
     .pipe(plugins.eslint.format());
 });
 
-gulp.task('test', ['lint']);
+gulp.task('mocha-test', () => {
+  return gulp.src(yargs.argv.file ? yargs.argv.file : 'test/test*.js', {read: false})
```

Comment:
> I am not a big enthusiast of naming files in a test folder testSomething but rather stick to test/engine.js to keep it (KI)SS. I am open for arguments against this simplification though.

The benefit of naming them test* is that you can write modules that aren't tests in the test/ directory (for example if there's some code shared between different tests).